packages/platform-web/src/node/features/html-render/lib/getPageMiddleware.ts

```diff
@@ -69,7 +100,25 @@ export async function getPageMiddleware(
 
     try {
       await pageHandler(req, res);
+      runPageMiddlewareTrace.setAttribute(
+        SHUVI_SERVER_RUN_PAGE_MIDDLEWARE.attrs.error.name,
+        false
+      );
+      runPageMiddlewareTrace.setAttribute(
+        SHUVI_SERVER_RUN_PAGE_MIDDLEWARE.attrs.statusCode.name,
+        res.statusCode
+      );
+      runPageMiddlewareTrace.stop();
     } catch (error) {
+      runPageMiddlewareTrace.setAttribute(
```

Comment:
It looks like it better to have a `setAttributes` method. e.g.

```
unPageMiddlewareTrace.setAttributes({
   'a': 1
   'b': 2
})
```

Reply:
added